python/lsst/meas/algorithms/detection.py

```diff
@@ -439,31 +474,79 @@ def plotPeaks(fps, ctype):
 
         return fpSets
 
-    def thresholdImage(self, image, thresholdParity, maskName="DETECTED"):
-        """!Threshold the convolved image, returning a FootprintSet.
-        Helper function for detect().
-
-        \param image The (optionally convolved) MaskedImage to threshold
-        \param thresholdParity Parity of threshold
-        \param maskName Name of mask to set
-
-        \return FootprintSet
+    def makeThreshold(self, image, thresholdParity):
+        """Make an afw.detection.Threshold object corresponding to the task's
+        configuration and the statistics of the given image.
+
+        Parameters
+        ----------
+        image : `afw.image.MaskedImage`
+            Image to measure noise statistics from if needed.
+        thresholdParity: `str`
+            One of "positive" or "negative", to set the kind of fluctuations
+            the Threshold will detect.
         """
         parity = False if thresholdParity == "negative" else True
-        threshold = afwDet.createThreshold(self.config.thresholdValue, self.config.thresholdType, parity)
+        threshold = afwDet.createThreshold(self.config.thresholdValue,
+                                           self.config.thresholdType, parity)
```

Comment:
I thought I'd only made these sorts of changes in code I touched, but git is more clever than I expected about seeing changes instead of blocks of new and old code; all of these are in a new function, but it's pretty similar to an old one I removed.